### ISSUE TITLE
testConfigure fails with latest nock version

### ISSUE THREAD

**Issue by marco-c** [mozilla/oghliner#211]
The `recreates existing GitHub token` test times out because:

```
{ [Error: Nock: No match for request get https://api.github.com/authorizations?page=1 ]
  defaultMessage: 'Internal Server Error',
  message: 'Nock: No match for request get https://api.github.com/authorizations?page=1 ',
  code: '500' }
```

Strangely, if I add:

```
    nock('https://api.github.com:443')
    .get('/authorizations')
    .query({"page":"1"})
    .reply(200, [{
      "id":22200031,
      "url":"https://api.github.com/authorizations/22200031",
      "app":{"name":"Oghliner token for " + slug,
      "url":"https://github.com/mozilla/oghliner",
      "client_id":"00000000000000000000"},
      "token":"",
      "hashed_token":"0000000000000000000000000000000000000000000000000000000000000000", // removed
      "token_last_eight":"00000000", // removed
      "note":"Oghliner token for " + slug,
      "note_url":"https://github.com/mozilla/oghliner",
      "created_at":"2015-09-16T20:08:41Z",
      "updated_at":"2015-09-16T20:08:41Z",
      "scopes":["public_repo"],
      "fingerprint":null
    }])
```

The test passes, but the after hook fails with:

```
Error: test finished with pending mocks: GET https://api.github.com:443/authorizations
```

**Comment by marco-c** [comment]
This change broke our tests: https://github.com/pgte/nock/commit/5eb5e2e7dc950c20f65b26fd6fed50dacec27cdf

**Comment by marco-c** [comment]
Fixed by #212.
